functions.php

```diff
@@ -69,33 +69,33 @@ function gesso_scripts() {
 
 	}
 
-  wp_register_script('gessocommon', get_template_directory_uri() . '/js/dist/common.min.js', array('jquery'), $gesso_version );
+  wp_register_script('gessocommon', get_template_directory_uri() . '/js/dist/common.min.js', array('jquery'), filemtime(get_template_directory_uri() . '/js/dist/common.min.js' ) );
   wp_enqueue_script('gessocommon');
 
-  wp_register_script('gessomodernizr', get_template_directory_uri() . '/js/libraries/modernizr.min.js', array('jquery'), $gesso_version ); // Modernizr
+  wp_register_script('gessomodernizr', get_template_directory_uri() . '/js/libraries/modernizr.min.js', array('jquery'), filemtime(get_template_directory_uri() .'/js/libraries/modernizr.min.js' ) ); // Modernizr
   wp_enqueue_script('gessomodernizr');
 
-  wp_register_script('gessodetailspolyfill', get_template_directory_uri() . '/js/libraries/details-element-polyfill.js', array('jquery'), $gesso_version ); // Details polyfill for IE/Edge
+  wp_register_script('gessodetailspolyfill', get_template_directory_uri() . '/js/libraries/details-element-polyfill.js', array('jquery'), filemtime(get_template_directory_uri() . '/js/libraries/details-element-polyfill.js' ) ); // Details polyfill for IE/Edge
```

Comment:
@sbrack8t FYI, when using the `filemtime` you need to use `get_template_directory()` to get the filesystem path and not  the URL.

Reply:
Updated